run.sh

```diff
@@ -16,4 +16,4 @@ hub pull-request \
     --draft \
     --message "${NEW_VERSION} RC" --message "${RELEASE_BODY}" \
     --base "main" \
-    --head "${BRANCH_NAME}";
\ No newline at end of file
+    --head "${NEW_VERSION}";
```

Comment:
nit: Should have final empty newline
```suggestion
    --head "${NEW_VERSION}";

```